polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -197,6 +208,13 @@ def send_torch_policy(
             If `blocking`, returns a list of RobotState objects. Otherwise, returns None.
 
         """
+        if use_mirror:
+            assert (
+                self.mirror_sim_robot
+            ), "Must call setup_mirror_for_forward before forward methods can be used on mirror sim"
+            return self.mirror_sim_robot.send_torch_policy(
+                torch_policy, blocking, timeout, use_mirror=False
+            )
```

Comment:
This block of code is repeated in `send_torch_policy` and `go_home`, and probably will also be in `move_to_joint_positions` and `move_to_ee_pose` in the future. Can you make a private method to deduplicate?

Reply:
Decorators are probably an elegant solution to this, if you're familiar with them.